core/templates/dev/head/components/question-directives/modal-templates/change-question-difficulty-modal.template.html

```diff
@@ -0,0 +1,19 @@
+<div class="modal-header">
+  <h3> Select new difficulty </h3>
+</div>
+<div class="modal-body select-skill-modal">
+  <div class="list-group">
+    <div class="list-group-item"  ng-repeat="skillWithDifficulty in linkedSkillsWithDifficulty track by $index">
+      <label class="list-group-item-heading" style="font-size: 1em;"><[skillWithDifficulty.getDescription()]></label>
+      <md-radio-group ng-model="skillWithDifficulty._difficulty">
```

Comment:
I think we should show the rubric along with the difficulty title, what do you think?

Reply:
I have done this for the initial question creation in #7770 , shall I do it in a follow up PR or maybe if this gets merged first, in the other PR? Since, maybe some code can be reused.

Reply:
Sounds good